### DISCUSSION TITLE
What is the right way to add alias and a utility for fish

### DISCUSSION THREAD

**Discussion by cmaspi** [fish-shell/fish-shell#8736] [Q&A] [answered]
I want to write a cli to-do list manager utility for fish, where should I add the alias  to call my fish script, and where should I put rest of the files related to the utility.

**Comment by ridiculousfish** [comment, marked as answer]
Why not just a .fish shell script? You can put it anywhere in $PATH, I have `~/bin/` in my path.

**Reply by cmaspi** [reply]
I have a bunch of other script which this main script would reference, I didn't want to put it directly in the Home directory, but ~/bin seems a nice place for that, I added my main script to ~/.local/bin and added other files to ~/.local/lib as I don't have a bin directory directly in the Home directory. Thanks for helping!
